test_suite/generic/test_generic.py

```diff
@@ -344,8 +344,14 @@ def test_sshd(self, host):
                 pytest.fail('ssh.service is not active')
 
             pass_auth_config_name = 'PasswordAuthentication'
+            pass_auth_config_file = '/etc/ssh/sshd_config'
 
-            if not host.file('/etc/ssh/sshd_config').contains(f'^{pass_auth_config_name} no'):
+            if host.system_info.distribution == 'fedora' and int(host.system_info.release) >= 38:
```

Comment:
Same as above, will this changes get into RHEL?

Reply:
Red Hat rebases cloud-init in RHEL, thus it might appear at some point in 8.x and 9.x. When exactly is a question for the maintainers of cloud-init, though.